tests/test_check_new_assets.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+import delayed_assert
+from scripts.utils.chain_model import Chain
+from tests.data.setting_data import chains
+
+task_ids = [
+    f'Test for {task.name}'
+    for task in chains
+]
+
+
+@pytest.mark.parametrize("chain", chains, ids=task_ids)
+class TestAssets:
+    def test_has_new_assets(self, chain: Chain):
+
+        added_assets = {}
+        for asset in chain.assets:
+            if asset['symbol'].upper() == 'AUSD':
+                added_assets.update({asset['symbol'].upper(): 'KUSD'})
+            else:
+                added_assets[asset['symbol'].upper()] = ''
```

Comment:
I would suggest to use dictionary comprehension in order to increase readability and make more uniq solution.
```suggestion
    # workaround to differences between asset names at runtime and in our configuration
    asset_mapping = {
        'AUSD': 'KUSD'
    }
    
    def test_has_new_assets(self, chain: Chain):
        
        chain_assets = {asset['symbol'].upper(): '' for asset in chain.assets}
        chain_assets.update(self.asset_mapping)
```